task/buildah-pr/0.1/buildah-pr.yaml

```diff
@@ -42,11 +45,13 @@ spec:
     outputs:
       - name: image
         type: image
-
+  workspaces:
+    - name: source
+      mountPath: /workspace/source
```

Comment:
for the `-pr` task, we don't use workspaces, on pipeline resource

Reply:
Fixed